keyboards/binepad/bn003/readme.md

```diff
@@ -0,0 +1,8 @@
+# BN003
+
```

Comment:
A lot of info is missing from here. Pic, reset instructions, flashing instructions, etc.
Please refer to https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs and complete this file.

Reply:
updated readme.md with macropad info